File: plugins/OrchidJavadoc/src/main/java/com/eden/orchid/javadoc/resources/ClassDocResource.java
package com.eden.orchid.javadoc.resources;

import com.eden.orchid.api.OrchidContext;
import com.sun.javadoc.ClassDoc;

public class ClassDocResource extends BaseJavadocResource {

    public ClassDocResource(OrchidContext context, ClassDoc classDoc) {
        super(context, classDoc.qualifiedName(), classDoc);
    }

}

File: plugins/OrchidJavadoc/src/main/kotlin/com/eden/orchid/javadoc/resources/ClassDocResource.kt
package com.eden.orchid.javadoc.resources

import com.copperleaf.javadoc.json.models.JavaClassDoc
import com.eden.orchid.api.OrchidContext

class ClassDocResource(
        context: OrchidContext,
        classDoc: JavaClassDoc
) : BaseJavadocResource(context, classDoc.qualifiedName, classDoc)
